app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2009 Nikias Bassen"
+LICENSE="GNU GPL v3"
```

Comment:
I've added all the mentioned authors to the COPYRIGHT section, let me know if this recipe needs to be further refactored.